java/vector/pom.xml

```diff
@@ -97,6 +103,58 @@
     </resources>
 
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <enableAssertions>true</enableAssertions>
+          <childDelegation>true</childDelegation>
+          <forkCount>${forkCount}</forkCount>
+          <reuseForks>true</reuseForks>
+          <systemPropertyVariables>
+            <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
+            <io.netty.tryReflectionSetAccessible>true</io.netty.tryReflectionSetAccessible>
+            <user.timezone>UTC</user.timezone>
+          </systemPropertyVariables>
+          <!-- Note: changing the below configuration might increase the max allocation size for a vector
+          which in turn can cause OOM. -->
+          <argLine>-Darrow.vector.max_allocation_bytes=1048576</argLine>
+        </configuration>
+        <executions>
+          <execution>
+            <!-- Skip default tests as they would have two allocation managers on classpath -->
```

Comment:
Can we simply remove this section?

Reply:
Hey @liyafan82, we can skip this section but we will run the tests 3 times then. One of the three will be with an unknown Allocator (it will be chosen at runtime depending on classpath). I thought it was a bit of a waste.

Reply:
Thanks for your clarification.

Reply:
When we run tests from IDE, it is also with an unknown allocator, right?

Reply:
yes, thats correct. We would have to set an env variable in the IDE to choose one or the other.

Reply:
would it work to remove the `skip=true` here and add ` <classpathDependencyExclude>org.apache.arrow:arrow-memory-unsafe</classpathDependencyExclude>`, so that the default test runs netty, then leave the additional `run-unsafe` section only?

Reply:
yes, that is quite a bit simpler. Apologies for misunderstanding originally.

I have made that change